src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -65,6 +66,34 @@ async function validate_input_types(type, action, argv) {
     }
 }
 
+/**
+ * validate_no_leading_space ensures that flags do not contain leading spaces
+ *  - if `argv_` contains more than two elements (type and action), it indicates an invalid flag value.
+ *
+ *
+ * ### CLI Parsing Behavior:
+ * When passing arguments via CLI, if a flag's value starts with a space or consists only of whitespace,
+ * the shell may treat it differently:
+ *  - Leading spaces - might be preserved when quoted (`--new_buckets_path=" abc/"`).
+ *  - Unquoted values with spaces - may cause incorrect parsing (`--new_buckets_path= abc/` is interpreted as an empty value).
+ *  - The parsed argument structure typically follows:
+ *      ```json
+ *      { _: [ "command", "action" ], flag1: "value1", flag2: "value2", ... }
+ *      ```
+ *  - The `_` array should contain only the command and action. If additional elements appear,
+ *    it suggests an issue with flag values (e.g., an incorrectly formatted or empty value).
+ *
+ * @param {string[]} argv_
+ * @throws {ManageCLIError}
+ */
+function validate_no_leading_space(argv_) {
+    // checking if 'argv_' contains more then 2 values (i.e, type and action)
+    if (argv_.length > 2) {
+        const details = `flag value is invalid - leading spaces detected in: ${JSON.stringify(argv_.slice(2))}, ensure flag values are properly formatted.`;
```

Comment:
checking argv_.length > 2 doesn't necassarly means that it's a leading spaces issue
this is a general issue that we found an unexpected "free" argument, not sure what is the term

for example - 
```
>  noobaa-cli account delete --name romy  blabla
```
[LOG] CONSOLE:: ROMY argv { _: [ 'account', 'delete', 'blabla' ], name: 'romy' }
I would rename the function and explain in the comments that there are several cases of this situation and one of them is leading spaces, and ofc throw a more general error and maybe add " or a leading spaces issues blablalal"

Reply:
Make sense, sure I will update it. Thanks